src/docs/truffle/reference/truffle-commands.md

```diff
@@ -341,18 +341,32 @@ Options:
 Download a Truffle Box, a pre-built Truffle project.
 
 ```shell
-truffle unbox <box_name> [--force]
+truffle unbox <box_name> [<destination_path>] [--force]
 ```
 
-Downloads a [Truffle Box](/boxes) to the current working directory. See the [list of available boxes](/boxes).
+Downloads a [Truffle Box](/boxes) to destination_path if provided. Truffle
+defaults to the current working directory if this argument is not provided.
 
-You can also design and create your own boxes!  See the section on [Truffle boxes](docs/truffle/getting-started/truffle-boxes) for more information.
+See the [list of available Truffle boxes](/boxes).
+
+You can also design and create your own boxes!  See the section on [Truffle boxes](docs/truffle/advanced/creating-a-truffle-box) for more information.
 
 Options:
 
 * `<box_name>`: Name of the Truffle Box. (required)
```

Comment:
Maybe this should be removed from the options section as it isn't really an option.

Reply:
That's a good point. Same applies to `run` and likely other commands on that page. Will merge this for now.